corehq/apps/app_manager/management/commands/migrate_case_search_labels.py

```diff
@@ -0,0 +1,29 @@
+from corehq.apps.app_manager.management.commands.helpers import (
+    AppMigrationCommandBase,
+)
+from corehq.apps.app_manager.util import get_correct_app_class
+
+
+class Command(AppMigrationCommandBase):
+    help = "One-time migration to replace CaseSearch command_label and again_label with " \
+           "search_label and search_again_label"
+
+    chunk_size = 20
+    include_builds = True
+    include_linked_apps = True
+
+    def migrate_app(self, app_doc):
+        should_save = False
+        for module in app_doc.get('modules', []):
+            if module.get('search_config'):
+                command_label = module['search_config'].get('command_label')
+                if command_label != module['search_config']['search_label']['label']:
```

Comment:
command_label would always be present with the default values for any app doc that has been saved, so just `if command_label` didn't seem useful here.
So thought of updating if they are different, which I believe would be always true as well, comparing default dict with empty. 
Not fully sure if the `if` here is even useful and we should just update app docs.
Open to thoughts on this if condition here.

Reply:
I realized later that `module['search_config']['search_label']['label']` would fail on old docs, so updated in a later commit to use `get`